mobify/documentation-theme
# Fix code examples to render correctly in Safari

| Before | After |
| --- | --- |
| <img width="800" alt="Screen Shot 2019-03-25 at 17 14 31" src="https://user-images.githubusercontent.com/734535/54962400-ecfc8d00-4f21-11e9-8f77-26e339c71eb1.png"> | <img width="801" alt="Screen Shot 2019-03-25 at 17 13 53" src="https://user-images.githubusercontent.com/734535/54962407-f38b0480-4f21-11e9-8b56-6d9f94e9146a.png"> |

## How to test

1. Inside this repo...
2. Run `npm link`
3. Run `npm run sass`
4. Switch to the `progressive-web-sdk` repo
5. Run `npm link @mobify/documentation-theme`
6. Run `npm run docs:dev`
7. Visit any component documentation page using Safari
8. **Verify** that the code examples' code sandboxes are rendering correctly (see screenshots above)
9. **Verify** the same thing in other browsers (Chrome, Firefox, iOS Safari, Android, etc)
